WordPress/Classes/Networking/Remote Objects/RemoteBlog.h

```diff
@@ -73,4 +73,9 @@
  */
 @property (nonatomic, strong) NSDictionary *capabilities;
 
+/**
+ * @details Parses details from a JSON dictionary.
+ */
+- (instancetype)initWithJSONDictionary:(NSDictionary *)json;
```

Comment:
Since we moved this out of the remote, I'd change the name (or at least the documentation) to reflect that this expects a JSON dictionary from the WordPress.com REST API. With WP-API in the horizon, let's try to be very explicit about what REST/JSON refers to.